app.js

```diff
@@ -44,7 +61,22 @@ app.get('/discover', discover.view);
 app.post('/discover', discover.getOrganizations);
 
 const dashboard = require('./routes/dashboard');
-app.get('/dashboard/:id/:page?', dashboard.view);
+app.get('/dashboard/:page?', isLoggedIn, dashboard.view);
+
+app.get('/auth/google',
+  passport.authenticate('google', { scope: ['profile'] }));
```

Comment:
If you can't keep every argument on the first like, I think it's generally preferable to do

```
app.get(
    argument1,
    argument2);
```

instead of

```
app.get(argument1,
    argument2);
```

Also, make sure that the arguments are +4 indentation. +2 is only ever used for "block-like" constructs, which start/end with ```{}[]```.